samples/A0000_Test.java_extracted.puml

```diff
@@ -0,0 +1,3 @@
+@startuml
```

Comment:
@arnaudroques: 
- How to manage sample? 
_(similar to [`pdiff`](https://github.com/plantuml/pdiff) project...)_
- That is an open question...
- A fundamental question will have to be asked on this subject...

Regards,
Th.